### DISCUSSION TITLE
Planning to deploy on Azure

### DISCUSSION THREAD

**Discussion by jbmarvin24** [prisma/prisma#3463] [Q&A] [answered]
Is Prisma supports to deploy on Azure Web Apps?

**Comment by ryands17** [comment, marked as answer]
Deploying to Azure Web Services would be the same as deploying any NodeJS application. You can find a tutorial on how to deploy a Node application [here](https://dev.to/bashirk/the-painless-way-to-deploying-your-nodejs-app-on-azure-part-1-14fi). Also one thing to make sure would be setting the `binaryTargets` in your `schema.prisma` as per the container that Azure uses so that there are no issues while querying as follows:

```prisma
generator client {
  provider        = "prisma-client-js"
// `native` is for your system and the other option must be added specific to the OS that Azure Web Apps uses.
  binaryTargets   = ["native", "linux-musl"]
}

datasource db {
  provider = "sqlite"
  url      = env("DB_URL")
}
```

You can find the list of binary targets [here](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema/prisma-schema-reference/#binarytargets-options) as per the Operating System. Let me know if that works.

**Reply by jbmarvin24** [reply]
Thanks for your answer and I try to deploy my test node application with Prisma and Azure Database for MySQL and it works perfectly.

**Reply by ryands17** [reply]
@gcbatman Awesome! It would be great if you could create a sample repo on GitHub with the steps so that everyone can take inspiration from that to deploy to Azure :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
